packages/dd-trace/src/appsec/remote_config/index.js

```diff
@@ -35,12 +35,14 @@ function enable (config) {
 function enableAsmData (appsecConfig) {
   if (rc && appsecConfig && appsecConfig.rules === undefined) {
     rc.updateCapabilities(RemoteConfigCapabilities.ASM_IP_BLOCKING, true)
+    rc.updateCapabilities(RemoteConfigCapabilities.ASM_USER_BLOCKING, true)
     rc.on('ASM_DATA', _asmDataListener)
   }
 }
 
 function disableAsmData () {
   if (rc) {
```

Comment:
this needs
`rc.updateCapabilities(RemoteConfigCapabilities.ASM_IP_BLOCKING, false)`
so it's consistent with the user blocking flag